src/test/kotlin/Generators.kt

```diff
@@ -0,0 +1,35 @@
+import io.kotlintest.properties.Gen
+
+class CellGen : Gen<Cell> {
+    override fun constants(): Iterable<Cell> = emptyList()
+
+    override fun random(): Sequence<Cell> =
+        generateSequence {
+            val random = Gen.bool().random().first()
+            if (random) Cell.Alive() else Cell.Dead()
+        }
```

Comment:
I think it should be something like:

```kotlin
Gen.bool().random().map { random -> if (random) Cell.Alive() else Cell.Dead() }
```

Reply:
This generator is not used anymore. All scenarios are tested using the universe generator that creates a fixed amount of alive or dead cells then shuffles the whole collection.